src/app/builder/builder.component.html

```diff
@@ -45,13 +45,20 @@
     </section>
 
     <section class="builder-step" *ngIf="feedUrl && !feedError" id="step-2">
-      <label for="">Select Episode</label>
-      <play-episode-picker (select)="onEpisodeSelect($event)" (invalidFeed)="onInvalidFeedNavigate($event)"
-      [feedUrl]="feedUrl" [selectedGUID]="episodeGuid"></play-episode-picker>
+      <fieldset>
+        <label for="updatingPlayer">Always Play Most Recent Episode</label>
```

Comment:
Note that your `for` isn't actually doing anything, since the checkbox input has no `id`.